packages/block-library/src/query-pagination/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit';
+
+const { name } = metadata;
+export { metadata, name };
+
+export const settings = {
+	title: __( 'Query Pagination' ),
+	parent: [ 'core/query' ],
```

Comment:
I guess same remark here as for the loop.

Reply:
Same answer :)